omod/src/main/resources/org/openmrs/module/fhir2/narratives/AllergyIntolerance.html

```diff
@@ -40,22 +40,22 @@
         <th:block th:unless="${fhirVersion} eq 'R4'">
```

Comment:
Are you sure about the logic here? R4 definitely has `clinicalStatus` and `verificationStatus` for AllergyIntolerance.

Reply:
@ibacher Yeah, but the type has been changed from `code` in R3 to `CodableConcept` in R4. Since `code` needs `th:text`, I had to separate out R3 and R4 here.

Reply:
Ok... but I actually prefer for these kinds of things that we'd do "${fhirVersion} eq 'DSTU3'" rather than this as future versions of FHIR are going to be based on R4. In other words, this is treating R4 as the exceptional case where as it's DSTU3 support that's the exceptional case.

Reply:
Okay, so I guess changing all R3->R4 diffs to "${fhirVersion} eq 'DSTU3'" would do right?

Reply:
Yes